### PR TITLE
Start documentation of CMD installation

### PR DESCRIPTION
**Pull request by enolfc**
# Summary 

Add CMD configuration

----

**Related issue :** #9

### CODE REVIEW

File: source/openstack.rst [hunk 1]
```diff
@@ -49,12 +49,14 @@ Optionally, **ooi (OpenStack OCCI Interface)** translates between OpenStack API
 Installation options
 ::::::::::::::::::::
 
-.. There are two options to install these components:
-   * Using the EGI FedCloud Appliance (recommended), which uses docker containers to bundle an OpenStack deployment of the corresponding services
-   * Using individual components.
+EGI distributes the integration components as:
+* A Virtual Appliance (VA) that users docker containers to bundle all of the
```
Comment (gwarf) [inline source/openstack.rst]: Some typos.

File: source/openstack.rst [hunk 2]
```diff
@@ -74,9 +76,13 @@ The appliance running at your OpenStack must:
 
 * Have enough disk space for handling the VM image replication (~ 100GB for ``fedcloud.egi.eu`` VO). By default these are stored at /image_data. You can mount a volume at that location.
 
-.. Individual Components
-   '''''''''''''''''''''
-   You can use the CMD repository to install packages for your distribution. Follow `the instructions for setuing up the repos <http://repository.egi.eu/category/os-distribution/cmd-os-1/>`_.
+CMD Packages
+''''''''''''
+
+CMD repositories provides packages that have gone through a quality assurance
```
Comment (gwarf) [inline source/openstack.rst]: s/\(provide\)s/\\1/

**Comment by gwarf** [comment]
@enolfc  can you please address comments?

**Comment by enolfc** [comment]
Changes rebased and comments addresed.

**Comment by gwarf** [comment]
Can you please rebase once again? :)

**Pull request merged**